.travis.yml

```diff
@@ -105,7 +105,7 @@ jobs:
 
     - stage: SimTests
       python: 3.7
-      env: SIM=ghdl
+      env: SIM=ghdl TOPLEVEL_LANG=vhdl
```

Comment:
This changes the test ID, so now it's not marked xfail any more

Reply:
Whoops. Will fix.